src/dfc.c

```diff
@@ -544,6 +544,11 @@ disp(struct list *lst, const char *fstfilter, const char *fsnfilter,
 			continue;
 		}
 
+		/* ignore /run mounts unless they're dangerously full */
+		if (!aflag && !strncmp(p->mntdir, "/run", 4) && p->perctused < 50) {
+			continue;
+		}
+
```

Comment:
Yeah, I guess it is fine as-is. It won't hurt platforms which are not concerned by this anyway.